modules/solid_mechanics/doc/content/source/materials/CombinedNonlinearHardeningPlasticity.md

```diff
@@ -0,0 +1,80 @@
+# Combined Nonlinear Hardening Plasticity
+
+!syntax description /Materials/CombinedNonlinearHardeningPlasticity
+
+## Description
+
+
+!include modules/solid_mechanics/common/supplementalRadialReturnStressUpdate.md
+
+In this model, the linear isotropic hardening function is given by $r = Q(1-exp(-bp))$, and kinematic
+hardening is governed by a backstress evolution equation of the form: $dX = Cdp - DXp$, where X is the backstress, and the two coefficients, C and D, represent the evolution of kinematic hardening.
```

Comment:
As we discussed, it would be good to include references to the two papers with that form of backstress evolution model.